src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Main.cs

```diff
@@ -114,16 +115,24 @@ bool Predicate(WindowsSetting found)
                     return true;
                 }
 
-                // Search for Area only by key char
-                if (found.Area.Contains(query.Search.Replace(":", string.Empty), StringComparison.CurrentCultureIgnoreCase)
-                && query.Search.EndsWith(":"))
+                if (!(found.Areas is null))
                 {
-                    return true;
-                }
+                    foreach (var area in found.Areas)
+                    {
+                        // Search for areas on normal queries.
+                        if (area.Contains(query.Search, StringComparison.CurrentCultureIgnoreCase)
+                        && (!query.Search.EndsWith(":")))
```

Comment:
Why do we need `(!query.Search.EndsWith(":")` here? Note, it wasn't part of this statement before.

Reply:
I thought it makes more clear that it is for queries without `:` at the end. But maybe it's obsolete.

Reply:
Let's remove it. If there are no particular reasons to keep it.